--- data/releases.list~	2008-05-13 08:41:15.000000000 -0400
+++ data/releases.list	2008-05-13 08:41:15.000000000 -0400
@@ -31,11 +31,11 @@
 #baseurl=http://download.fedora.redhat.com/pub/fedora/linux/development/$basearch/os
 
 # Not available yet
-#[Fedora 9 (Sulphur)]
-#stable=True
-#version=9
-#mirrorlist=http://mirrors.fedoraproject.org/mirrorlist?repo=fedora-9&arch=$basearch
-#installmirrorlist=http://mirrors.fedoraproject.org/mirrorlist?path=pub/fedora/linux/releases/9/Fedora/$basearch/os
+[Fedora 9 (Sulphur)]
+stable=True
+version=9
+mirrorlist=http://mirrors.fedoraproject.org/mirrorlist?repo=fedora-9&arch=$basearch
+installmirrorlist=http://mirrors.fedoraproject.org/mirrorlist?path=pub/fedora/linux/releases/9/Fedora/$basearch/os
 
 [Fedora 8 (Werewolf)]
 # We don't consider this stable, as it's not a supported preupgrade target
