diff -up preupgrade-0.9.8/data/releases.list.f10beta preupgrade-0.9.8/data/releases.list
--- preupgrade-0.9.8/data/releases.list.f10beta	2008-10-02 18:47:36.000000000 -0400
+++ preupgrade-0.9.8/data/releases.list	2008-10-02 18:47:47.000000000 -0400
@@ -34,11 +34,11 @@ version=999
 mirrorlist=http://mirrors.fedoraproject.org/mirrorlist?repo=rawhide&arch=$basearch
 #baseurl=http://download.fedora.redhat.com/pub/fedora/linux/development/$basearch/os
 
-#[Fedora 10 (Cambridge) Beta]
-#stable=False
-#preupgrade-ok=True
-#version=9.92
-#mirrorlist=http://mirrors.fedoraproject.org/mirrorlist?path=pub/fedora/linux/releases/test/10-BetaFedora/$basearch/os
+[Fedora 10 (Cambridge) Beta]
+stable=False
+preupgrade-ok=True
+version=9.92
+mirrorlist=http://mirrors.fedoraproject.org/mirrorlist?path=pub/fedora/linux/releases/test/10-Beta/Fedora/$basearch/os
 
 # You can't reliably preupgrade to these.
 # They're only here so we know when they've gone EOL.
