diff -up preupgrade-0.9.8/preupgrade-gtk.py.fix-resume preupgrade-0.9.8/preupgrade-gtk.py
--- preupgrade-0.9.8/preupgrade-gtk.py.fix-resume	2008-10-02 18:46:30.000000000 -0400
+++ preupgrade-0.9.8/preupgrade-gtk.py	2008-10-02 18:46:46.000000000 -0400
@@ -325,7 +325,9 @@ class PreUpgradeGtk(PreUpgradeController
                 r = questionDialog(self.ui.assistant,
                     "Would you like to resume your upgrade to %s?" % resuming)
                 if not r:
-                    os.unlink(preupgrade.runfile)
+                    # clear out old caches / boot files / etc
+                    print "Clearing data from upgrade to %s" % resuming
+                    self.pu.clear_incomplete_run()
                 else:
                     # Act like we just hit the 'Apply' button
                     if resuming not in releases:
diff -up preupgrade-0.9.8/preupgrade/__init__.py.fix-resume preupgrade-0.9.8/preupgrade/__init__.py
--- preupgrade-0.9.8/preupgrade/__init__.py.fix-resume	2008-10-02 18:46:41.000000000 -0400
+++ preupgrade-0.9.8/preupgrade/__init__.py	2008-10-02 18:46:55.000000000 -0400
@@ -552,6 +552,17 @@ class PreUpgrade(yum.YumBase):
             pass
         return releasename
 
+    def clear_incomplete_run(self):
+        '''Clear all the saved data for an incomplete run.'''
+        release = self.resuming_run()
+        assert release != None
+        self.setup(release)
+        self.setup_update_repo()
+        self.remove_repo_cache()
+        self.remove_boot_files()
+        os.unlink(runfile)
+        self.clear_update_repo()
+
     def retrieve_critical_boot_files(self):
         # relative tmp FIXME XXX
         
