=== modified file 'preupgrade/__init__.py'
--- preupgrade/__init__.py	2008-05-02 14:50:34 +0000
+++ preupgrade/__init__.py	2008-05-13 18:10:41 +0000
@@ -369,7 +369,7 @@
         # min image
         # example:
         #   http://download.fedora.redhat.com/pub/fedora/linux/development/i386/os/.treeinfo
-        tif = self.distrepo.grab.urlopen('/.treeinfo')
+        tif = self.instrepo.grab.urlopen('/.treeinfo')
         #cracked out ppc issues, etc
         archlist = [self.conf.yumvar['basearch']]
         if isParaVirt():

